lldb/test/API/lang/swift/expression/weak_generic_self/TestSwiftWeakGenericSelf.py

```diff
@@ -0,0 +1,18 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import unittest2
+
+
+class TestSwiftWeakGenericSelf(TestBase):
+    @swiftTest
+    def test(self):
+        """Test that a weakly captured generic class type can be used in expression evaluation"""
```

Comment:
This was a fun one! 🙂

```suggestion
        """Confirms that an expression evaluation works with a generic class type within a closure that weakly captures it"""
```